bookdown/01-introduction.Rmd

```diff
@@ -22,3 +27,41 @@ Other _building blocks_ include:
 - Resampling methods
 
 These additional blocks are provided by our extension packages such as `r gh_pkg("mlr-org/mlr3tuning")`, `r gh_pkg("mlr-org/mlr3pipelines")` or `r gh_pkg("mlr-org/mlr3featuresel")`.ing.
+
+## Equivalent elements from `mlr`
```

Comment:
I would simply call it "transition table" or similar. 
Not everything will be equivalent in mlr3 and the list will also contain a lot of missing items.